django/db/migrations/serializer.py

```diff
@@ -155,11 +155,6 @@ def serialize(self):
             raise ValueError("Cannot serialize function: lambda")
         if self.value.__module__ is None:
             raise ValueError("Cannot serialize function %r: No module" % self.value)
-        # Python 3 is a lot easier, and only uses this branch if it's not local.
```

Comment:
@andrewgodwin, can you confirm this removal? I'm not sure if some test coverage might be missing considering the comment implies Python 3 does use this branch.